checkpoint_ngfw/help.md

```diff
@@ -554,16 +562,16 @@ This action is used to create a rule to block traffic.
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|action|string|Drop|True|Action to take|['Accept', 'Drop', 'Ask', 'Inform', 'Reject', 'User Auth', 'Client Auth', 'Apply Layer']|
-|destination|string|None|False|Destination network object name|None|
-|discard_other_sessions|boolean|True|True|Discard all other user sessions. This can fix errors when objects are locked by other sessions|None|
-|layer|string|Network|True|Layer to add this rule to|None|
-|list_of_services|[]string|None|False|List of services to block e.g. ["AOL", "SMTP"]|None|
-|name|string|None|True|Rule name|None|
-|position|string|top|True|Position in the list of rules. e.g. top, bottom, 15|None|
-|source|string|None|False|Source network object name|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|action|string|Drop|True|Action to take|['Accept', 'Drop', 'Ask', 'Inform', 'Reject', 'User Auth', 'Client Auth', 'Apply Layer']|Drop|
+|destination|string|None|False|Destination network object name|None|192.168.2.1|
+|discard_other_sessions|boolean|True|True|Discard all other user sessions. This can fix errors when objects are locked by other sessions|None|True|
+|layer|string|Network|True|Layer to add this rule to|None|Network|
+|list_of_services|[]string|None|False|List of services to block e.g. ["AOL", "SMTP"]|None|['AOL', 'SMTP']|
+|name|string|None|True|Rule name|None|Malicious IP Addresses|
+|position|string|top|True|Position in the list of rules. e.g. top, bottom, 15|None|1|
```

Comment:
```suggestion
|position|string|top|True|Position in the list of rules.  Can be a number or one of the words "top" or "bottom"|None|1|
```
If this field can be a number or string, I think listing all valid string values would be helpful.

Reply:
Good find @tylerslijboom-r7